lib/internal/webstreams/readablestream.js

```diff
@@ -140,6 +140,19 @@ const kError = Symbol('kError');
 const kPull = Symbol('kPull');
 const kRelease = Symbol('kRelease');
 
+let releasedError;
+let releasingError;
+
+function lazyReadableReleasedError() {
+  releasedError ??= new ERR_INVALID_STATE.TypeError('Reader released');
```

Comment:
I watched your live stream analyzing this optimization, you told me that maybe you would use `Object.freeze` to prevent others from modifying the error that will be used in streams, do you have any reason not to use it?